cogs/gameplay.py

```diff
@@ -80,6 +87,52 @@ async def game(self, ctx: commands.Context, level: str):
         await view.end_game()
         await view.response.edit(view=view)
 
+    @commands.command(aliases=["q"])
+    async def quiz(self, ctx: commands.Context):
+        """IUFI Quiz"""
+        # Fetch the user data
+        user = await func.get_user(ctx.author.id)
+
+        # If the cooldown is still in effect, inform the user and exit
+        if (retry := user.get("cooldown", {}).setdefault("quiz_game", 0)) > time.time():
+            return await ctx.reply(f"{ctx.author.mention} your quiz is <t:{round(retry)}:R>", delete_after=10)
+
+        # Get the game state for the quiz game, or set to default values if not found
+        game_state = user.get("game_state", {}).get("quiz_game", {})
+        attempted = game_state.get("attempted", {
+            "first_time": 0,
+            "times": 0
+        })
+
+        # If the user has attempted more than 5 times in a day, inform them and exit
+        if (time.time() - attempted["first_time"] < 1440) and attempted["times"] >= 5:
```

Comment:
instead of hardcoding max number of times here, can we keep it in GAME_SETTINGS or something like that for quiz config?

Also 1440 is seconds right? which would mean 1440 seconds = 24 mintues. it should be 86400 for a day